invenio_mail/ext.py

```diff
@@ -82,6 +84,8 @@ def init_app(self, app):
         if app.config.get('MAIL_SUPPRESS_SEND', False) or app.debug:
             email_dispatched.connect(print_email)
         app.extensions['invenio-mail'] = self
+        # only used so we can inherit from "base.html"
+        app.register_blueprint(blueprint)
```

Comment:
Should we completely leave out the templates and let Invenio-Theme define them instead?